include/ceed/ceed.h

```diff
@@ -180,10 +180,10 @@ CEED_EXTERN int CeedErrorImpl(Ceed, const char *, int, const char *, int,
 /// Use nonstandard ternary to convince the compiler/clang-tidy that this
 /// function never returns zero.
 #  define CeedError(ceed, ecode, ...)                                     \
-  (CeedErrorImpl((ceed), __FILE__, __LINE__, __func__, (ecode), __VA_ARGS__) ?: (ecode))
+  (CeedErrorImpl((ceed), __FILE__, __LINE__, __func__, (ecode), __VA_ARGS__) ? (ecode) : (ecode))
 #else
 #  define CeedError(ceed, ecode, ...)                                     \
-  CeedErrorImpl((ceed), __FILE__, __LINE__, __func__, (ecode), __VA_ARGS__) ?: (ecode)
+  CeedErrorImpl((ceed), __FILE__, __LINE__, __func__, (ecode), __VA_ARGS__) ? (ecode) : (ecode)
```

Comment:
Once upon a time, the `__clang__` variant was to appease clang and the other was a standard ternary. I think the idea was to let `CeedErrorImpl` decide to return something different. What we have now is just comma operator; do the first thing and return the second.